babel.config.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
We're now using SWC for compiling, not Babel. Why did you add this? CC @ibacher who may be able to clarify when and how we are using SWC vs Babel.

Reply:
@brandones babel config was not added by me but rather in between the original PR and now, @ibacher made the [migration](https://github.com/openmrs/openmrs-esm-patient-management/commit/f2440cd5f3fc3df92127e128d98b209df00de6cf). I did had to extend babel config for test plugin transform-require-context - in order to allow rendering patient-banner app and resolve webpack's require.context function. In the course of re-basing to master seem this file got added back again... I have cleaned this and remove any dependencies to babel... thanks

Reply:
Currently the intention is that babel is only used for [this transformation](https://github.com/openmrs/openmrs-esm-core/blob/master/packages/tooling/webpack-config/src/optimize.ts).